examples/backends/sglang/launch/xpu/disagg_encoder_xpu.sh

```
@@ -0,0 +1,309 @@
+    DYN_VLLM_EMBEDDING_TRANSFER_MODE=nixl-read \
+    ENABLE_ENCODER_CACHE=0 \
+    VISION_ENCODE_SERIALIZE=1 \
+    NIXL_USE_CPU_HOST_MEMORY=1 \
+    python3 -m dynamo.sglang \
+        --model "$MODEL_PATH" \
+        --enable-multimodal \
+        --multimodal-encode-worker \
+        --chat-template qwen2-vl \
+        --dtype auto \
+        --kv-cache-dtype auto \
+        --mem-fraction-static $MEM_FRACTION \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)